main.go

```diff
@@ -96,6 +102,14 @@ func main() {
 		config.fromName = config.fromNameOnError
 		config.message = config.messageOnError
 		config.color = config.colorOnError
+		config.notify = config.notifyOnError
+	}
+
+	// Convert notify "true", "false" to "1", "0" to match API expectation
+	if config.notify == "true" {
```

Comment:
Changing value in main is not properly managed, could you please modify `step.yml`'s `value_options` in both notify & notify_on_error fields instead of `true; false` to `1; 0`?